duoStrength.js

```diff
@@ -529,9 +530,10 @@ function addStrengths(strengths)
 			`
 				position: absolute;
 				display: inline-block;
-				width: 85%;
+				width: 100%;
 				height: 1em;
 				left: 0;
+				transform: inherit;

```

Comment:
As the transform style of the `strengthBarHolder` isn't being applied this doesn't have an effect.

```suggestion
```